### PR TITLE
Add UT for DefaultServiceIPRange

### PR DESCRIPTION
**Pull request by wackxu**
**What this PR does / why we need it**:
1. Add UT for DefaultServiceIPRange
2. Remove unused TODO  because the code has already move out of the genericapiserver package

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by wackxu** [comment]
@justinsb @janetkuo @derekwaynecarr  Please take a look at this PR, it needs your approve, thanks.

**Comment by wackxu** [comment]
@justinsb @janetkuo @derekwaynecarr Please take a look at this PR, it needs your approve, thanks.

**Comment by wackxu** [comment]
@justinsb @janetkuo @derekwaynecarr Please take a look at this PR, it needs your approve, thanks.

File: pkg/master/services_test.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+			resultIpNet: net.IPNet{},
+			resultIp:    "<nil>",
+			stopError:   fmt.Errorf("The service cluster IP range must be at least %d IP addresses", 8),
+		},
+	}
+	for _, testCase := range testCases {
+		ipNet, ip, err := DefaultServiceIPRange(testCase.ipNet)
+
+		if testCase.stopError != nil && err != nil {
+			continue
```
Comment (justinsb) [inline pkg/master/services_test.go]: You could check the value of `err.Error()` here (it returns a string, it's the only function on the error interface)

File: pkg/master/services_test.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+		}
+
+		if testCase.stopError == nil && err != nil {
+			t.Errorf("case %v, unexpected error: %v", testCase.name, err)
```
Comment (justinsb) [inline pkg/master/services_test.go]: I think you missed the case where stopError != nil && err == nil.

Maybe check the stopError != nil case separately from the stopError == nil case.

**Comment by wackxu** [comment]
@justinsb Thanks for your review. Done, PTAL

**Comment by justinsb** [comment]
Looks like tests failing with:

```
I0526 20:27:20.878] --- FAIL: TestDefaultServiceIPRange (0.00s)
I0526 20:27:20.878] 	services_test.go:87: want ipNet: {10.0.0.0 ffffff00}, get: {10.0.0.0 ffffff00}
```

(not sure why!)

**Pull request closed without merging**